code/modules/reagents/reagent_containers/syringes.dm

```diff
@@ -145,7 +145,7 @@
 
 /obj/item/reagent_containers/syringe/update_overlays()
 	. = ..()
-	if(reagents?.total_volume)
+	if(reagents?.total_volume && !istype(src, /obj/item/reagent_containers/syringe/crude))
```

Comment:
You should instead move out the body of this proc (everything except `..()`) into its own proc that `/crude` then overrides to be a no-op.

Reply:
Please confirm that I did what you asked.